StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsGenericInfoScreen.swift

```diff
@@ -60,6 +63,7 @@ struct FinancialConnectionsGenericInfoScreen: Decodable {
                 enum BodyEntryType: String, Codable {
                     case text = "text"
                     case image = "image"
+                    case bullets = "bullets"
```

Comment:
[backend bullets type and definition](https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/text/data/server_driven_u_x/generic_info_body.rb#L39-43)